samples/openapi3/client/petstore/python-experimental/test/test_animal_farm.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+
+"""
+    OpenAPI Petstore
+
+    This spec is mainly for testing Petstore server and contains fake endpoints, models. Please do not use this for any other purpose. Special characters: \" \\  # noqa: E501
+
+    The version of the OpenAPI document: 1.0.0
+    Generated by: https://openapi-generator.tech
+"""
+
+
+from __future__ import absolute_import
+import sys
+import unittest
+
+import petstore_api
+try:
+    from petstore_api.model import animal
+except ImportError:
+    animal = sys.modules[
+        'petstore_api.model.animal']
+from petstore_api.model.animal_farm import AnimalFarm
+
+
+class TestAnimalFarm(unittest.TestCase):
+    """AnimalFarm unit test stubs"""
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def testAnimalFarm(self):
+        """Test AnimalFarm"""
+        # FIXME: construct object with mandatory attributes with example values
+        # model = AnimalFarm()  # noqa: E501
+        pass
```

Comment:
How about adding a test here like
animal = [animal.Animal(blah)]
model = AnimalFarm(animal)
assert model[0] === animal

- How about adding a test in test/test_fake_api.py where we send an instance of AnimalFarm to the endpoint that accepts it?

- How about adding a deserialization test like we have [here](https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/python-experimental/tests/test_deserialization.py#L86) to show that our code can receive back array models and deserialize them correctly?

Reply:
@spacether I will go back to my fake example. There are too many issues with `Animal` models.